runtime/compiler/env/VMJ9Server.hpp

```diff
@@ -298,6 +298,7 @@ class TR_J9SharedCacheServerVM: public TR_J9ServerVM
 
    // replacing calls to isAOT
    virtual bool       canUseSymbolValidationManager() override                 { return true; }
+   virtual bool       canTrackAOTDependencies() override                       { return true; }
```

Comment:
Should this be made 'false' until we support this feature for JITServer?

Reply:
JITServer AOT compilations should be fine, I thought, just not JITServer AOT cache compilations